boilerplate/e2e/firstTest.spec.js

```diff
@@ -0,0 +1,14 @@
+describe('Example', () => {
```

Comment:
Let's add a comment at the top with the URL to Detox's guide on matchers and whatnot, so they know where to look for more information on how to build tests.

Reply:
There's a link in the README, but putting it here next to the tests is a good idea.